lib/roo_on_rails/railties/database.rb

```diff
@@ -1,7 +1,7 @@
 module RooOnRails
   module Railties
     class Database < Rails::Railtie
-      if ActiveRecord::VERSION::MAJOR >= 4
+      if defined?(ActiveRecord) && ActiveRecord::VERSION::MAJOR >= 4
```

Comment:
`on_load` is well documented in Rails 5+ and seems to exist before. I guess it's worth a try.